bin/index.js

```diff
@@ -20,7 +20,7 @@ require('yargs')
     proxy(proxyConfig);
   })
   .command('deploy', 'Deploys your theme to AEM instance', (yargs) => {}, function (argv) {
-    shell.exec('dotenv -- cross-var curl -u %AEM_USER%:%AEM_PASS% -d \'prefixPath=/%GIT_ORG%/%GIT_REPO%/%GIT_ARTIFACT_ID%\' -d \'githubToken=%GIT_TOKEN%\' %URL%/conf/%SITE%.updatetheme.html');
+    shell.exec('dotenv -- cross-var curl -d \'site=%SITE%\' -d \'artifact=%GIT_ARTIFACT_ID%\' -d \'salt=%SALT%\' -d \'hash=%HASH%\' %URL%/aem/update.theme.html');
```

Comment:
Is `hash` the token? If yes, I'd keep `GIT_TOKEN ` for now until we tackle the ticket where we rename the vars. wdyt?